File: bindings/php/src/ForgeLib.php
<?php

declare(strict_types=1);

namespace ForgeLib;

final class ForgeLib
{
    public const STATUS = 'metadata-only';
    public const REPOSITORY = 'https://github.com/isala404/forge';

    public static function notice(): string
    {
        return 'Forge PHP bindings are planned. Use Rust, Node.js, or Python today.';
    }
}

File: bindings/elixir/lib/forgelib.ex
defmodule Forgelib do
  @moduledoc """
  Metadata for the future first-party Forge Elixir binding.
  """

  @status "metadata-only"
  @repository "https://github.com/isala404/forge"

  def status, do: @status
  def repository, do: @repository

  def notice do
    "Forge Elixir bindings are planned. Use Rust, Node.js, or Python today."
  end
end
